purchase_product_weight/models/purchase.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+
+from odoo import models, fields, api
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = 'purchase.order.line'
+
+    weight_total = fields.Float(string='Total weight')
+    product_weight = fields.Float(
+        related='product_id.weight',
+        string='Product Weight')
+
+    @api.onchange('product_id', 'product_weight', 'product_qty')
```

Comment:
I am just wondering why do we need to have product_weight here at all?
At the time of onchange, there shouldn't be change of changing the weight, or is it?
(I think normally, changing of weight can be for computed field)

Reply:
I removed it, thanks!
(I would like to have a computed field which user can modify by hand and if user goes on product from PO and alter the weight, the field was recomputed, but I don't know if it is possible)